client/components/convert-to-blocks/index.jsx

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ConvertToBlocksDialog extends Component {
+	static propTypes = {
+		close: PropTypes.func,
+		showDialog: PropTypes.bool,
+		confirmConversion: PropTypes.func,
+	};
+
+	shouldComponentUpdate( nextProps ) {
+		return this.props.showDialog !== nextProps.showDialog;
```

Comment:
Do we really need this? These conditions are usually required on components connected to the state, in order to avoid the component to be updated every time any value of the state changes. Since this component is stateless, I think we can rely on React to handle when the component should be updated.

Reply:
I completely agree. I'll pull it out :)